### DISCUSSION TITLE
I am not able to verify as a student.

### DISCUSSION THREAD

**Discussion by Charles-IX** [community/community#75499] [GitHub Education]
### Hi everyone,

I had difficulty in verifying as a student.
My school, Zhengzhou University of Light Industry is a verified school on GitHub education and I have a email address followed by @email.zzuli.edu.cn, but I still can't finish the request.

The original error is pasted as follows:

> Your application cannot be reviewed until you fix the following:
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.
Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.
You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

Yes, I am using an VPN but you know the Internet condition in China mainland -- I can't even surf GitHub if I turn that off.
What can I do? 

Thanks for everyone who volunteer his or her time to help a newbie like me in this community.

**Comment by AidenJamesYt** [comment]
It appears that you are facing two main issues:

GitHub Education's location verification system is unable to detect your physical location due to your VPN usage. As a result, the system is requesting that you provide documentation indicating virtual learning.

Your GitHub billing information does not match the name on your academic affiliation document. You need to ensure that your legal name is entered exactly as it appears on your academic documentation.

To resolve these issues, you can try the following:

Temporarily disable your VPN when verifying your student status. This will allow GitHub Education to detect your physical location and validate your enrollment at Zhengzhou University of Light Industry.

Verify that your GitHub billing information matches your legal name as it appears on your academic affiliation document. Double-check for any typos or discrepancies.

If you have already disabled your VPN and verified your billing information, but you are still encountering errors, you can contact GitHub Education support. They can further investigate the issue and provide assistance if necessary.

if you do contact github support directly (i recommend opening a ticket), remember to explain the situation clearly to GitHub Education support, including your VPN usage and the reasons behind it, as well as any steps you've already taken to resolve the issue.
